docs/src/pages/index.tsx

```diff
@@ -142,6 +142,17 @@ export default function Home(
           additional help.
         </p>
       </Section>
+      <Section sectionClassName={styles.sectionBackground}>
+        <p>
+          <a
+            href="https://github.com/open-olive/loop-development-kit/blob/main/LICENSE"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Terms & Conditions
```

Comment:
This isn't a terms and conditions, it's a license.

I think we should have 2 links here. Something like this. I didn't include them here, but you can keep the target and rel attributes.
```html
<p>
  <a href="https://github.com/open-olive/loop-development-kit/blob/main/LICENSE">Loop Development Kit Open Source License</a>
    |
  <a href="https://olive.page.link/olive-helps-terms">Olive Helps Terms &amp; Conditions</a>
</p>
```

Reply:
Looks good now. Thanks!